index.rst

```diff
@@ -381,78 +441,18 @@ For other components, see the exception section below.
-
-An example output is, ``heads/develop-0-gc89ef1a``.
-The repository branch (or tag) name forms the first part of the branch description.
-This first part contain individual identifiers and can change rapidly.
-It may take any form necessary to convey the appropriate information.
-The last 7 characters (``c89ef1a``) is the hash of the commit of repository, so all configuration files in that repo correspond to the same hash.
-Users can find this commit by navigating to the repository on github, searching for the commit hash, then
-clicking on the "commits" section of the search results, as shown in :ref:`the screenshot below <fig-commit-tracing>`.
-
-.. figure:: /_static/tracing_a_commit_on_github.jpg
-    :name: fig-commit-tracing
-
-    Using the ``version`` output in the ``configurationApplied`` event, it is possible to traceback the repo to the configuration that was loaded.
-
-Once we have identified the hash of the commit file we want to reload, we can do that without having to make any changes to the currently deployed ts_config package.
-If we simply want to use the default site-specific configuration for a given CSC, we can specify the commit hash with a preceding colon (``:``) as follows:
-
-.. code-block:: python
-
-    from lsst.ts.observatory.control import ATCS
-
-    atcs = ATCS()
-
-    await atcs.start_task
-
-    # ATAOS must be in STANDBY state for this to work. All other CSCs will
-    # use their default configurations
-    await atcs.enable(configurationOverride={'ATAOS': ':c89ef1a'})
-
-If we also want to specify an override file then we insert the filename before the colon (``:``) as shown below:
-
-.. code-block:: python
-
-    await atcs.enable(configurationOverride={'ATAOS': 'simple_algorithm.yaml:c89ef1a'})
-.. _section-on-the-fly-config:
 
 On-the-Fly Configuration Changes
---------------------------------
+================================
 
-During the process of creating a new configuration (:ref:`section-configuration-creating-a-new`) or during a commissioning/engineering run, it may be necessary to make a new configuration available to a running CSC for testing without rebuilding/re-deploying the component.
-In these cases, the user should also create a Jira ticket (or work out of an existing ticket) to document the occurrence.
+During the process of creating a new configuration (see also `Modifying or Creating a New Configuration`_) or during a commissioning/engineering run, it may be necessary to make a new configuration available to a running CSC. 
+Normally, new configurations are only made available when rebuilding/re-deploying the component and associated configuration repository, but this is not always a feasible solution.
+Therefore, in cases where on-the-fly configuration changes are required, this procedure should be followed.
+Like the other procedures, the user should create a Jira ticket (or work out of an existing ticket) to document the occurrence.
```

Comment:
I suggest making this the first step. Also end the line above with a colon delete the line below (reducing repetition).

Reply:
I re-worded. The first step of the referenced procedure is to create a jira ticket.